coverage/jacoco.gradle

```diff
@@ -0,0 +1,20 @@
+
+if (Boolean.getBoolean("coverageEnabled")) {
+    apply plugin: 'jacoco'
```

Comment:
We already having something like this as a buildSrc plugin. If the gradle property `jacoco.enabled` is `true`.

https://github.com/deephaven/deephaven-core/blob/v0.37.4/buildSrc/src/main/groovy/io.deephaven.java-test-conventions.gradle#L5-L9

https://github.com/deephaven/deephaven-core/blob/v0.37.4/buildSrc/src/main/groovy/io.deephaven.java-jacoco-conventions.gradle